build.go

```diff
@@ -132,11 +132,17 @@ func (m buildModule) getBuild(buildID uint) (Build, error) {
 	var build Build
 	if err := m.Database.
 		Where(&Build{BuildID: buildID}).
+    Preload(database.BuildFields.TestResultSummaries).
```

Comment:
nit: indentation bo-bo

```suggestion
		Preload(database.BuildFields.TestResultSummaries).
```